.github/workflows/posix.yml

```diff
@@ -62,8 +59,17 @@ jobs:
             pandas_testing_mode: "deprecate"
             test_args: "-W error"
             name: "Numpy Dev"
+          # Test lowest supported pyarrow
+          - env_file: actions-38.yaml
+            pattern: "not single_cpu"
+            pyarrow_version: 5
```

Comment:
There was a desire to have more pyarrow version coverage, so I think we should still add a 3.8 build with pyarrow 7 (`"not single_cpu"` and `"single_cpu"`)